packages/core/components/QuerySidebar/tutorials/ShareView.ts

```diff
@@ -9,5 +9,5 @@ export const SHARE_VIEW_TUTORIAL = new Tutorial("Sharing current view (URL)")
     .addStep({
         targetId: Tutorial.ADD_QUERY_BUTTON_ID,
         message:
-            'Start a new query from a URL by clicking the New Query + button then "Import from URL" then pasting or typing a URL into the textbox. Press Enter to load the query.',
+            'If in a web browser paste it into your URL bar in a new tab otherwise start a new query from a URL by clicking the New Query + button then "Import from URL" then pasting or typing a URL into the textbox. Press Enter to load the query.',
```

Comment:
nit: 'If in a web browser, paste the link into your URL bar in a new tab. Otherwise, start a...'
Also maybe instead of 'then "Import from URL" then pasting or typing a URL into the textbox' could be 'then paste or type a URL into the "Paste URL" textbox'

Reply:
Fixed in #b1bbfe1